atm.js

```diff
@@ -1,3 +1,191 @@
+/*********
+Pseudo Code At Bottom
+**********/
+
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+/*******
+CHECKINGS
+********/
+
+  var checkingTotal = 0;
+  if (checkingTotal === 0) {
+    $("#checking .balance").css("background", "rgb(232, 63, 63)")
+  }
+
+  $("#checking .deposit").on("click", function (evt) {
+    evt.preventDefault();
+    var depositAmount = $("#checking .input").val()
+    if (isNaN(depositAmount)) {
+      $("#checking .balance").html("<p class='smallText'>Please enter an amount</p>");
+      $("#checking .balance").css("background", "rgb(232, 63, 63)");
```

Comment:
Love your attention to detail though. You're really accounting for all the scenarios here.